subprojects/base-services/src/main/java/org/gradle/internal/io/LinePerThreadBufferingOutputStream.java

```diff
@@ -16,14 +16,16 @@
 
 package org.gradle.internal.io;
 
+import org.gradle.internal.SystemProperties;
+
 import java.io.IOException;
 import java.io.PrintStream;
 import java.util.Locale;
 
 public class LinePerThreadBufferingOutputStream extends PrintStream {
     private final TextStream handler;
     private final ThreadLocal<PrintStream> stream = new ThreadLocal<PrintStream>();
-
+    String lineSeparator = SystemProperties.getInstance().getLineSeparator();
```

Comment:
Can this be moved into the getStream method below or can this be made to look like the other fields (private final...)

Reply:
fixed. I think it doesn't work within getStream()